201-api-management-logs-to-moesif-using-eventhub-webapp/azuredeploy.json

```diff
@@ -0,0 +1,324 @@
+            }
+        },
+        "azureAppServiceSku": {
+            "type": "string",
+            "defaultValue": "B1",
+            "minLength": 2,
+            "metadata": {
+                "description": "The instance / SKU name for Azure App Service eg: B1, B2, S1, P1V2. Note F1 and D1 plan are not supported as they do not support 'alwaysOn'"
+            }
+        },
+        "dnsNamePrefix": {
+            "type": "string",
+            "defaultValue": "[concat('moesiflog', uniqueString(resourceGroup().id))]",
+            "minLength": 6,
+            "metadata": {
+                "description": "A prefix that will be added to created resource names and DNS URLs. Allowed characters: alphabets and numbers only. Resulting name must be maximum 24 characters (storage account maximum)"
+            }
+        },
+        "_artifactsLocation": {
+            "type": "string",
+            "defaultValue": "https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/201-api-management-logs-to-moesif-using-eventhub-webapp/",
+            "metadata": {
+                "description": "The base URL where templates are located. Should end with trailing '/'"
+            }
+        },
+        "_artifactsLocationSasToken": {
+            "type": "securestring",
+            "defaultValue": "",
+            "metadata": {
+                "description": "The sasToken required to access _artifactsLocation.  When the template is deployed using the accompanying scripts, a sasToken will be automatically generated."
+            }
+        }
+    },
+    "functions": [],
+    "variables": {
+        "moesifSessionToken": "optional",
+        "moesifApiVersion": "v1",
+        "apiManagementLoggerId": "moesif-log-to-event-hub",
+        "eventHubSendPolicyName": "[concat(parameters('dnsNamePrefix'), '-send-policy')]",
+        "eventHubListenPolicy": "[concat(parameters('dnsNamePrefix'), '-listen-policy')]",
+        "azStorageTemplUri": "[uri(parameters('_artifactsLocation'), concat('nested/microsoft.storage/storageaccounts.json', parameters('_artifactsLocationSasToken')))]",
+        "azEventHubTemplUri": "[uri(parameters('_artifactsLocation'), concat('nested/microsoft.eventhub/namespaces.json', parameters('_artifactsLocationSasToken')))]",
+        "azApiMgmtLoggerInstallTemplUri": "[uri(parameters('_artifactsLocation'), concat('nested/microsoft.apimanagement/service/loggers.json', parameters('_artifactsLocationSasToken')))]",
+        "azAppServicePlanTemplUri": "[uri(parameters('_artifactsLocation'), concat('nested/microsoft.web/serverfarms.json', parameters('_artifactsLocationSasToken')))]",
+        "azAppServiceTemplUri": "[uri(parameters('_artifactsLocation'), concat('nested/microsoft.web/sites.json', parameters('_artifactsLocationSasToken')))]",
+        //"azAppServiceWebJobTemplUri": "[uri(parameters('_artifactsLocation'), concat('nested/microsoft.web/sites/extensions.json', parameters('_artifactsLocationSasToken')))]",
```

Comment:
remove commented lines of code

Reply:
There are two sections of commented out code in this PR. The reason why it is commented out, is because of MSDeploy failing WebJob deployment. Please see my comments in README response to your question.
However, if an end user were to deploy this template (copy paste) into Azure portal custom deployment, despite ms deploy failure, they will be able to use the webjob. 

I intended to phase the PR in two steps: Without MSDeploy (this PR) and after resolving issues with MSDeploy submit a second PR. I am happy to act on your feedback.

Reply:
That seems fine - but for now remove it if we're not supporting MSDeploy in this sample and we can publish.  Then you can submit another PR with MSDeploy and we can see if we can debug what's happening there.

Reply:
fixed. Will do for future PR. Thx.